disdrodb/data_transfer/scripts/download_disdrodb_archive.py

```diff
@@ -0,0 +1,79 @@
+# -----------------------------------------------------------------------------.
+# Copyright (c) 2021-2022 DISDRODB developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------.
+####################################################################
+## Wrapper to download disdrodb archives by command lines ##
+####################################################################
+import click
+
+
+@click.command()
+@click.option(
+    "--disdrodb_dir",
+    required=True,
+    type=str,
+    show_default=True,
+    default="",
+    help="Raw data folder path (eg : /home/user/DISDRODB/Raw). Is compulsory.",
+)
+@click.option(
+    "--data_sources",
+    multiple=True,
+    type=str,

```

Comment:
This `click` options should could be defined as a single function (see `click_l0_stations_options` https://github.com/ltelab/disdrodb/blob/main/disdrodb/l0/l0_processing.py#L1179) and to be reused also for the upload code. 
They should also be tested that pass the arguments as expected when called from terminal ... 
@regislon I feat that if `multiple=True` it returns always a list (of str)  and maybe `parse_arg_to_list` does not deal well with that (it expects a str). `parse_arg_to_list` should be tested for all possible cases !!! 
Before, I assumed that in the terminal the user typed i.e `--data_sources='EPFL NCAR' ` using the space delimiter to provide multiple data_sources. With multiple now one can say `--data_sources EPFL --data_sources NCAR` right?
Now both cases are supported? And `--data_sources 'EPFL GPM' --data_sources NCAR` does crash right? 
Maybe we can simply keep the old behaviour and do not use `multiple=True`. Up to you.

Reply:
let's be consistant over the project. I've removed the multiple = True